### PR TITLE
263 Feature Request: Scenario Template & Contribution Guide

### PR DESCRIPTION
**Pull request by TeneBrae93**
#### Overview of Changes
- Created a scenario template with basic resources, output, and whitelisting to provide a starting place for CloudGoat contributors. I also provided comments explaining some of the main things to keep in mind. 
- Updated the Contribution Guidelines on the main README to help people get started with contributing and linked them to the scenario template as a starting point

### CODE REVIEW

File: scenarios/scenario_template/cheat_sheet.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# [SERVICE]_Secrets Walkthrough
```
Comment (andrew-aiken) [inline scenarios/scenario_template/cheat_sheet.md:1]: Was `_Secrets` meant to be left in? Not sure if this was carried over from sns_secrets

File: README.md [hunk 2]
```diff
@@ -328,13 +328,30 @@ For features, much the same applies! Be specific in your request, and make sure
 
 Contributions to CloudGoat are greatly appreciated. If you'd like to help make the project better, read on.
 
-1. Python code in CloudGoat should generally follow Python's style conventions, favoring readability and maintainability above all.
-2. Follow good git practices: use pull requests, prefer feature branches, always write clear commit messages.
-3. CloudGoat uses `black` and `flake8` - Python syntax and style linters - If you're going to commit code for CloudGoat, ensure that first `flake8`, and then `black` are both run on all Python files in `core/python/` and on `cloudgoat.py`. `black`'s decisions take priority over `flake8`'s. Both of these are commented out in the `core/python/requirements.txt` file since normal users don't need them.
-4. CloudGoat code should always use the BSD 3-clause license.
+1. **Creating a New Scenario**:
```
Comment (andrew-aiken) [inline README.md:331]: 😍

**Review by andrew-aiken — CHANGES_REQUESTED** [review]
Overall looks good

There already is a "hidden" template directory
https://github.com/RhinoSecurityLabs/cloudgoat/tree/master/core/terraform/example-scenario

File: scenarios/scenario_template/cheat_sheet.md (hunk 1 above)
Comment (TeneBrae93) [reply scenarios/scenario_template/cheat_sheet.md]: Oh good catch, I missed that one. Just fixed it

File: README.md (hunk 2 above)
Comment (TeneBrae93) [reply README.md:331]: Hahaha glad you like it!

**Comment by TeneBrae93** [comment]
Good catch @andrew-aiken on the existing template (didn't even realize that) - and neither did the people I talked to at Rhino when we discussed this. It's also pretty light on details (doesn't show whitelisting a public resource like an EC2 which is a specific problem we ran into with a community-submitted scenario - the EC2 is public on the internet). 

I added a new commit that just removes this old one entirely, and puts this new one in the scenario folder. I think this makes it easier for people to find; also by adding it to the main README's contribution guidelines, this one should be less hidden. 

Finally, I noticed some things I missed (manifest.yml in particular), so I added that to the new scenario template.

**Review by andrew-aiken — APPROVED** [review]
Awesome
Thanks for improving the readme, should make it a lot easier for people to create scenarios

**Review by DaveYesland — APPROVED** [review]

**Review by nobodynate — APPROVED** [review]
Looks good!

**Pull request merged**